.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+# Description
+<!-- add a short summary if necessary; mention issue numbers -->
+
+# Checklist
+<!-- add a "X" inside the brackets to confirm the point -->
+* [ ] Make sure changes are covered by existing or new tests.
+* [ ] Make sure local test run is green for at least one Python version.
+* [ ] Create a file in `src/towncrier/newsfragments/` for our changelog.
```

Comment:
For towncrier we have `NEWS`.

I tried to inform users that the content of the file should not be a commit message, but it should contain important public information for other users of the project.

Feel free to repharse

```suggestion
* [ ] Create a file in `src/towncrier/newsfragments/` describing your change and that will be included in the public release notes.
```